docs/code/MlNetCookBook.md

```diff
@@ -285,11 +285,11 @@ Or by creating a data model for it:
 ```csharp
 private class AdultData
 {
-    [LoadColumn("0", "10"), ColumnName("Features")]
-    public float FeatureVector { get; }
+    [LoadColumn(0, 10), ColumnName("Features")]
+    public float FeatureVector { get; set;}
 
     [LoadColumn(11)]
-    public float Target { get; }
+    public float Target { get; set;}
```

Comment:
Same here. Not sure if having `get` and `set` is a convention but I'd remove them for shorter bug equivalent code.
```suggestion
    public float Target;
```